validation/validation_test.go

```diff
@@ -11,17 +11,21 @@ import (
 	"github.com/mrunalp/fileutils"
 	"github.com/opencontainers/runtime-tools/generate"
 	"github.com/satori/go.uuid"
+	"github.com/stretchr/testify/assert"
 )
 
 var (
 	runtime = "runc"
 )
 
 func init() {
-	runtime = os.Getenv("RUNTIME")
+	runtimeInEnv := os.Getenv("RUNTIME")
+	if runtimeInEnv != "" {
+		runtime = runtimeInEnv
```

Comment:
This looks like a non-WIP improvement to me.  Can we move it into its own PR to get it landed more quickly?

Reply:
https://github.com/opencontainers/runtime-tools/pull/448 is created.